Source/Engine/Core/Math/Quaternion.cs

```diff
@@ -975,7 +975,7 @@ public static void LookAt(ref Vector3 eye, ref Vector3 target, ref Vector3 up, o
         /// <param name="target">The camera look-at target.</param>
         /// <param name="up">The camera's up vector.</param>
         /// <returns>The created look-at quaternion.</returns>
-        public static Quaternion LookAt(Vector3 eye, Vector3 target, Vector3 up)
+        public static Quaternion LookAt(Vector3 eye, Vector3 target, Vector3 up  = Vector3.Up)
```

Comment:
The default value must be a compile time constant, that's why it's not possible to set a custom value, C# limitation.
There is a workaround using nullable value but it's not very clean imo.

Reply:
fixed